src/lib/db.js

```diff
@@ -26,22 +27,35 @@ const TABLE_NAMES = [
   'L_LU_TRANSFER_STATUS'
 ]
 
+const MAX_RETRY = 1000
```

Comment:
Is 1000 retries a reasonable number? Would it make sense to let transactions fail after fewer attempts?

I wonder if a high number of retries can be counterproductive. For example, failing retries from different transactions might stack up and eventually slow down the database.